thirty-three-across/thirty-three-across-htb.js

```diff
@@ -379,12 +396,15 @@ function ThirtyThreeAcrossHtb(configs) {
             },
             user: {
                 ext: {
-                    consent: gdprConsent.consentString
+                    consent: privacyStats.tcf.consentString
                 }
             },
             regs: {
                 ext: {
-                    gdpr: gdprConsent.gdpr
+                    // NOTE: using CCPA version for now, or move to ttx.tcf.version/ttx.ccpa.version in the future
+                    version: privacyStats.ccpa.version,
```

Comment:
why not use `ttx` extension directly to send gdpr and ccpa version?

Reply:
ok, let me move to ttx extension